tools/internal_ci/helper_scripts/prepare_build_linux_perf_rc

```diff
@@ -21,7 +21,13 @@ ulimit -c unlimited
 
 # Performance PR testing needs GH API key and PR metadata to comment results
 if [ -n "$KOKORO_GITHUB_PULL_REQUEST_NUMBER" ]; then
-  sudo apt-get install -y jq
+  retry=0
```

Comment:
this is ok as a short term fix but I'd like something cleaner and more maintainable in the long run.
let's add a TODO to find a better way to do this.

another nit: doest `sudo apt-get install -y jq || (sleep 5; sudo apt-get install -y jq) || (sleep 5; sudo apt-get install -y jq)` look more readable?

Reply:
the current code makes it easier to adjust the number of retries and the time between each retry